chainexchange/chainexchange.go

```diff
@@ -0,0 +1,27 @@
+package chainexchange
+
+import (
+	"context"
+
+	"github.com/filecoin-project/go-f3/gpbft"
+)
+
+type Key []byte
```

Comment:
maybe make it a string so it can be used in caches and for comparisons?

Reply:
The reason I haven't is because of avoiding invalid strings once this gets encoded as CBOR (in upcoming partial message that includes it).